pages/ksphere/kommander/1.2/clusters/attach-cluster/federated-addons/index.md

```diff
@@ -31,25 +33,25 @@ If you want to federate a cluster by creating a new `KommanderCluster` object in
+| ------------------------------------ | -------------------- | -------------------------------------------------------- | ------------------------------------- |
+| dashboard-kommander-override         | True                 |                                                          | False                                 |
+| elasticsearch                        | False                | `kommander.mesosphere.io/federate-elasticsearch`         | True                                  |
+| elasticsearch-curator                | False                | `kommander.mesosphere.io/federate-elasticsearch-curator` | True                                  |
+| elasticsearchexporter                | False                | `kommander.mesosphere.io/federate-elasticsearchexporter` | True                                  |
+| fluentbit                            | False                | `kommander.mesosphere.io/federate-fluentbit`             | True                                  |
+| kibana                               | False                | `kommander.mesosphere.io/federate-kibana`                | True                                  |
+| kube-oidc-proxy-kommander            | True                 |                                                          | False                                 |
+| opsportal-addons-kommander-overrides | True                 |                                                          | False                                 |
+| prometheus                           | True                 | `kommander.mesosphere.io/federate-prometheus`            | True                                  |
+| prometheusadapter                    | True                 | `kommander.mesosphere.io/federate-prometheusadapter`     | True                                  |
+| reloader                             | True                 |                                                          | True                                  |
+| traefik-forward-auth-kommander       | True                 |                                                          | False                                 |
 
 ## Federated cluster addons
 
-| Name | Federated by default | Federation label | Only federated on non-Konvoy clusters |
-|------------|----------------|----------------|----------------|
-- cert-manager | True | | True |
-- kubecost | True | | False |
-- traefik | True | | True |
+| Name           | Federated by default | Federation label | Only federated on non-Konvoy clusters |
+| -------------- | -------------------- | ---------------- | ------------------------------------- |
+| - cert-manager | True                 |                  | True                                  |
+| - kubecost     | True                 |                  | False                                 |
+| - traefik      | True                 |                  |  True                                 |
```

Comment:
since these are in a table now, we can take out the `-` in front of each addon

Reply:
that change actually just was my prettier tooling, I didnt change it manually 🙈 
seems like the markup was invalid before. so the question is: should it be a table or list?

Reply:
> that change actually just was my prettier tooling, I didnt change it manually 🙈
> seems like the markup was invalid before. so the question is: should it be a table or list?

This looks pretty good as a table, although the federation label column is empty? For 508 compliance, should have something in every box.